tests/app/questionnaire/test_questionnaire_store_updater.py

```diff
@@ -87,14 +87,7 @@ def test_save_answers_data_with_default_value(self):
         )
         self.questionnaire_store_updater.update_answers(form)
 
-        assert self.answer_store.add_or_update.call_count == 1
-
-        created_answer = self.answer_store.add_or_update.call_args[0][0]
-        assert created_answer.__dict__ == {
-            'answer_id': answer_id,
-            'list_item_id': None,
-            'value': default_value,
-        }
+        assert self.answer_store.add_or_update.call_count == 0
```

Comment:
This test doesn't assert what the method name suggests. Shouldn't we be checking the content of the answer store only?

Reply:
`test_default_answers_are_not_saved`?